api/AudioContext.json

```diff
@@ -55,7 +55,7 @@
             },
             {
               "version_added": "15",
-              "version_removed": "44",
+              "version_removed": "15",
```

Comment:
43 should be correct according to the mapping, I believe. Added and removed both in 15 makes no sense.